cmd/mr_note_test.go

```diff
@@ -23,6 +23,19 @@ func Test_mrCreateNote(t *testing.T) {
 	}
 
 	require.Contains(t, string(b), "https://gitlab.com/lab-testing/test/merge_requests/1#note_")
+
+	// Escaped sequence like the following one was already a problem while
+	// retrieving the argument list
+	cmd = exec.Command(labBinaryPath, "mr", "note", "lab-testing", "1",
+		"-m", "{\"key\": \"value\"}")
+
+	b, err = cmd.CombinedOutput()
+	if err != nil {
+		t.Log(string(b))
+		t.Fatal(err)
+	}
+
+	require.Contains(t, string(b), "https://gitlab.com/lab-testing/test/merge_requests/1#note_")
```

Comment:
Suggest wrapping this in a subtest, and linking to the issue ( https://github.com/zaquestion/lab/issues/376 )

Reply:
Sure thing. Will do.
Thanks!